src/core/runResponse.ts

```diff
@@ -0,0 +1,30 @@
+import { IBundleRouter } from '../bundle/bundleRouter';
+import { createServerProcess, IServerProcess } from '../devServer/server';
+import { createServerEntry } from '../production/module/ServerEntry';
+import { Context } from './context';
+
+export const createRunResponse = async (ctx: Context, router: IBundleRouter) => {
+  const bundles = await router.writeBundles();

```

Comment:
if it's `createRunResponse` we shouldn't put writeBundles here

Reply:
refactored